_component_design/cards.md

```diff
@@ -99,36 +64,36 @@ Small and large spacing values can be used for card variations but should not be
 <div class="hxRow"  markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Spacing
+### Composition
```

Comment:
i just noticed that there are spacing blocks on this composition image. we need to remove them.

Reply:
Also, the redline specs should really be in the style graphic. HOWEVER, since the style graphic s really talking about the grid layout of cards, i'm fine leaving it in the composition image.